src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -83,7 +88,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         private CSharpParseOptions(CSharpParseOptions other) : this(
-            languageVersion: other.LanguageVersion,
+            languageVersion: other.SpecifiedLanguageVersion,
```

Comment:
No, that would copy the effective language version into the specified language version field of the copy. That would be incorrect.